Dockerfile

```diff
@@ -117,7 +117,7 @@ RUN curl -L http://http.debian.net/debian/pool/main/libc/libcap2/libcap2_2.22.or
 RUN cd /linux-kernel && \
     make INSTALL_HDR_PATH=/tmp/kheaders headers_install && \
     cd / && \
-    git clone http://git.code.sf.net/p/aufs/aufs-util && \
+    git clone https://github.com/Distrotech/aufs-util.git && \
```

Comment:
According to the [announcement](https://webcache.googleusercontent.com/search?q=cache:3BwaFSfCUL4J:sourceforge.net/p/aufs/mailman/aufs-users/thread/18905.1431883360%40jrobl/+&cd=2&hl=nl&ct=clnk&gl=nl) all repo's moved to @sfjro except for the aufs-util repo :|

How certain are we that @distrotech is the right use/place to get the sources?
See [here](https://webcache.googleusercontent.com/search?q=cache:e_JMQzhpl4IJ:aufs.sf.net/+&cd=1&hl=nl&ct=clnk&gl=nl) for a cached version of the overview.

Also, Sourceforge is having an outage, it's not like they announced they would be shutting down like Google Code. So is it necessary to move away from Sourceforge?